packages/flutter/lib/src/widgets/text.dart

```diff
@@ -449,7 +521,7 @@ class Text extends StatelessWidget {
       maxLines: maxLines ?? defaultTextStyle.maxLines,
       strutStyle: strutStyle,
       textWidthBasis: textWidthBasis ?? defaultTextStyle.textWidthBasis,
-      textHeightBehavior: textHeightBehavior ?? defaultTextStyle.textHeightBehavior,
+      textHeightBehavior: textHeightBehavior ?? defaultTextHeightBehavior.textHeightBehavior ?? defaultTextStyle.textHeightBehavior,
```

Comment:
This is really unfortunate, i.e. I would have expected: 
```dart
 textHeightBehavior ?? defaultTextStyle.textHeightBehavior ?? defaultTextHeightBehavior.textHeightBehavior;
```

Would it be possible to change DefaultTextStyle.textHeightBehavior to be null by default, and have null mean `const TextHeightBehavior()`?

Reply:
Discussed offline, but yes I'll adopt this pattern.